<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Xenomai_periodic_timer_checker</class>
 <widget class="QMainWindow" name="Xenomai_periodic_timer_checker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Xenomai periodic timer checker</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(70, 70, 70);
color: rgb(180, 180, 180);</string>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(70, 70, 70);
color: rgb(180, 180, 180);</string>
    </property>
    <property name="text">
     <string>Stop1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic_xeno">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>420</width>
      <height>92</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(150, 150, 150);</string>
    </property>
    <property name="text">
     <string>mode</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_checkermode">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>79</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 170, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>141</y>
      <width>302</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(150, 150, 150);</string>
       </property>
       <property name="text">
        <string>sampling time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="checking_sampling_time">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(150, 150, 150);
border-color: rgb(0, 0, 255);
gridline-color: rgb(85, 0, 255);</string>
       </property>
       <property name="text">
        <string>100000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: rgb(150, 150, 150);</string>
       </property>
       <property name="text">
        <string>nano second</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>201</y>
      <width>275</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color: rgb(150, 150, 150);</string>
       </property>
       <property name="text">
        <string>Max error</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="checking_sampling_time_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">color: rgb(150, 150, 150);</string>
       </property>
       <property name="text">
        <string>nano second</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>182</x>
      <y>238</y>
      <width>300</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">color: rgb(150, 150, 150);</string>
       </property>
       <property name="text">
        <string>Average error</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="checking_sampling_time_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">color: rgb(150, 150, 150);</string>
       </property>
       <property name="text">
        <string>nano second</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_stop2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(70, 70, 70);
color: rgb(180, 180, 180);</string>
    </property>
    <property name="text">
     <string>Stop2</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>554</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
